### PR TITLE
Add aws registry method

### PR DESCRIPTION
**Pull request by dvonessen**
What does this PR change? What problem does it solve?
-----------------------------------------------------

Adds to `gcip.addons.container.registries.Registry` the AWS staticmethod to allow getting ECR registry urls.

Was the change discussed in an issue?
------------------------------------------------------------

Checklist
---------

- [x] I have read the [Contribution Guidelines](https://github.com/dbsystel/gitlab-ci-python-library/blob/main/CONTRIBUTING.md)
- [x] I have enabled [maintainer edits for this PR](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork)
- [x] I have added tests for all changes in this PR
- [x] I have added api documentation for the changes
- [x] I added a changelog entry to the unreleased section with the PR number. [Changelog](https://github.com/dbsystel/gitlab-ci-python-library/blob/main/CHANGELOG.md)
- [x] Linting is done, and linting jobs are successfull
- [x] I'm done, this Pull Request is ready for review

### CODE REVIEW

File: gcip/addons/aws/helper.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import os
+
+import boto3
+
+session = boto3.Session()
+sts = session.client("sts")
```
Comment (ThomasSteinbach) [inline gcip/addons/aws/helper.py:6]: `session` is never used

```suggestion
sts = boto3.Session().client("sts")
```

File: gcip/addons/aws/helper.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    2. From response of `sts.get_caller_identity()` function call with an initialized `boto3.client`
+
+    Returns:
+        str: AWS account id.
+    """
+    if os.environ.get("AWS_ACCOUNT_ID"):
+        return os.environ["AWS_ACCOUNT_ID"]
+
+    sts_response = sts.get_caller_identity()
+    return sts_response["Account"]
```
Comment (ThomasSteinbach) [inline gcip/addons/aws/helper.py:24]: `sts_response` is never used but as return value

```suggestion
    return sts.get_caller_identity()["Account"]
```

**Review by ThomasSteinbach — CHANGES_REQUESTED** [review]
The unit test comparison files seems to be missing.

File: gcip/addons/aws/helper.py (hunk 1 above)
Comment (dvonessen) [reply gcip/addons/aws/helper.py:6]: Accepted

File: gcip/addons/aws/helper.py (hunk 2 above)
Comment (dvonessen) [reply gcip/addons/aws/helper.py:24]: Accepted

**Comment by dvonessen** [comment]
@ThomasSteinbach there are no comparison files, all methods and functions do not emit GitLab CI config. They are used as a base to build Jobs or Sequences.

**Review by ThomasSteinbach — APPROVED** [review]

**Pull request merged**